lib/addons/adot/index.ts

```diff
@@ -25,22 +27,37 @@ const defaultProps = {
 @supportsALL
 export class AdotCollectorAddOn extends CoreAddOn {
 
+    private namespace: string;
```

Comment:
we try to avoid member variable other than the props, since blueprints can be cloned an reused. 
You can drop line 39 and just use in the deploy method  `this.coreAddOnProps.namespace`

Reply:
Updated the code, was able to test successfully as well

```
 k get all -n adot                                                                                                                                                [10/11/23| 8:47AM]
NAME                                                READY   STATUS    RESTARTS   AGE
pod/otel-collector-amp-collector-748d886bdd-kp6bp   1/1     Running   0          11h
pod/otel-collector-xray-collector-c76d86886-gxmq4   1/1     Running   0          11h

NAME                                               TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)             AGE
service/otel-collector-amp-collector-monitoring    ClusterIP   172.20.126.84   <none>        8888/TCP            11h
service/otel-collector-xray-collector              ClusterIP   172.20.205.87   <none>        4317/TCP,4318/TCP   11h
service/otel-collector-xray-collector-headless     ClusterIP   None            <none>        4317/TCP,4318/TCP   11h
service/otel-collector-xray-collector-monitoring   ClusterIP   172.20.96.232   <none>        8888/TCP            11h

NAME                                            READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/otel-collector-amp-collector    1/1     1            1           11h
deployment.apps/otel-collector-xray-collector   1/1     1            1           11h

NAME                                                      DESIRED   CURRENT   READY   AGE
replicaset.apps/otel-collector-amp-collector-748d886bdd   1         1         1       11h
replicaset.apps/otel-collector-xray-collector-c76d86886   1         1         1       11h
```